pkgs/top-level/all-packages.nix

```diff
@@ -9302,6 +9302,10 @@ let
 
   radius = callPackage ../servers/radius { };
 
+  rake = callPackage ../development/tools/build-managers/rake {
+#    ruby = ruby_2_1;
```

Comment:
Unless this is intended to serve as documentation of some sort, please remove it.